source/options.css

```diff
@@ -23,6 +23,10 @@ html {
 		--link-color: #8ab4f8;
 		--input-border-color: rgba(255, 255, 255, 0.3);
 	}
+
+	.is-firefox {
+		--background-color: #201f24;
+	}
```

Comment:
This doesn’t need JavaScript, there’s a media query just for firefox just below. 

Also, do you think that `html, body {background: none !important}` works in either browser? I’d prefer that if you can test it